### PR TITLE
Don't retry on fail in wait_for_kueue_available.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
## Fixes / Features
- Don't retry on fail in wait_for_kueue_available.

## Testing / Documentation
Testing details.

- [ y/n ] Tests pass
- [ y/n ] Appropriate changes to documentation are included in the PR

### CODE REVIEW

**Comment by mbobrovskyi** [comment]
cc: @pawloch00

**Comment by pawloch00** [comment]
does it work with private cluster?

**Comment by mbobrovskyi** [comment]
No, it doesn’t fix the issue with the private cluster. However, we don’t need to wait 30 minutes.

File: src/xpk/core/kueue.py [hunk 1]
```diff
@@ -201,7 +201,7 @@ def wait_for_kueue_available(args: Namespace) -> int:
       ' --for=condition=available --timeout=5m'
```
Comment (pawloch00) [inline src/xpk/core/kueue.py:201]: Please remove hardcoded timeout value please.

**Review by pawloch00 — APPROVED** [review]

**Comment by mbobrovskyi** [comment]
cc @IrvingMg

**Review by IrvingMg — APPROVED** [review]

**Pull request merged**